pkgs/by-name/th/thc-secure-delete/package.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, stdenvNoCC # x86_64-darwin and aarch64-darwin cant find C compiler
+, fetchFromGitHub
+, pkg-config
+, gcc
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "thc-secure-delete";
+  version = "3.1.1";
+
+  src = fetchFromGitHub {
+    owner = "gordrs";
+    repo = "thc-secure-delete";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-hvWuxFkzhOSCplPtyjRtn36bIk6KdPBcpr3lAmiAyfE=";
+  };
+
+  build-systems = [ pkg-config gcc ];
+
+  buildPhase = ''
+    substituteInPlace Makefile --replace "gcc" "${gcc}/bin/gcc"
+    make
+  '';
+
+  installPhase = ''
+    runHook preInstall
+    install -Dm755 srm $out/bin
+    install -Dm755 sfill $out/bin
+    install -Dm755 sswap $out/bin
+    install -Dm755 sdmem $out/bin
+    install -Dm755 sfill.1 $out/share/man/man1/sfill.1.gz
```

Comment:
Those files are not actually gzipped, are they?

Reply:
yes they are